src/components/ui/BaseBox.vue

```diff
@@ -0,0 +1,12 @@
+<template>
+  <div class="base-box">
+    <slot></slot>
+  </div>
+</template>
+
+<style lang="scss">
+.base-box {
+  border-radius: 0.625rem;
```

Comment:
I wouldn't use rem for border radius. It'd be better to add variables and use var here :)